app/scripts/models/reliers/relier.js

```diff
@@ -101,6 +102,19 @@ define([
     },
 
     /**
+     * Check if the relier is Sync for Firefox Desktop
+     */
+    isChooseWhatToSyncWeb: function (uniqueUserId, locationParams, able) {
+      var abData = {
+        uniqueUserId: uniqueUserId,
+        // the window parameter will override any ab testing features
+        forceChooseWhatToSyncMode: Url.searchParam('force_choose_what_to_sync', locationParams)
```

Comment:
Can `force_choose_what_to_sync` be imported in fetch, and then retrieved using `this.get('force_choose_what_to_sync')` here? That'd eliminate the requirement to pass in `locationParams`.